streaming/base/converters/dataframe_to_mds.py

```diff
@@ -201,12 +159,10 @@ def write_mds(iterator: Iterable):
         else:
             raise RuntimeError('TaskContext.get() returns None')
 
-        if isinstance(mds_path, str):  # local
-            output = os.path.join(mds_path, f'{id}')
-            out_file_path = output
+        if mds_path[1] == '':
```

Comment:
there's duplicate naming between `mds_path` defined in this `write_mds` function and the returned `mds_path` from the larger `dataframeToMDS` function. could you change the naming so this is more clear?

Reply:
They are actually the same variables picked to the executor.